src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -21,16 +21,46 @@ import Foundation
 import ExposureNotification
 import UIKit
 
-protocol ExposureSummaryProvider: AnyObject {
-	typealias Completion = (ENExposureDetectionSummary?) -> Void
-	func detectExposure(completion: @escaping Completion)
+
+/**
+A RiskProvider is a **Publisher** providing risk calculation results to its **subscribers** (`RiskConsumer`)
+To obtain an instance of the `RiskProvider`, your class needs to implement the protocol `RequiresAppDependencies` that has the default implementation offer the single instance of `RiskProvider`.
+
+~~~
+class MyVc : UIViewController, RequiresAppDependencies{
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		let riskProvider = self.riskProvider
+	}
 }
+~~~
+
+To get the result of risk calculation, you need the following steps:
+~~~
+// 1. Create an instance of RiskConsumer
+let riskConsumer = RiskConsumer()
+
+// 2. Attach the callback block to handle the result
+riskConsumer.didCalculateRisk = {risk in
+    // bla bla bla
+}
+
+// 3. Obtain the instance of RiskProvider and attach the RiskConsumer to the RiskProvider
+riskProvider.observeRisk(riskConsumer)
 
+// 4. Ask for the result
```

Comment:
General question: Why are some comment blocks done with `/*    */' and some with `///` ?

Reply:
In general, they're equivalent. For multilines, /** */ is better. /// is suitable for single line doc.